modules/nf-core/art/illumina/main.nf

```diff
@@ -0,0 +1,71 @@
+process ART_ILLUMINA {
+
+    tag "$meta.id"
+    label 'process_single'
+
+    // WARN: Version information not provided by tool on CLI. Please update version string below when bumping container versions.
+    conda "bioconda::art=2016.06.05"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/art:2016.06.05--h589041f_9':
+        'quay.io/biocontainers/art:2016.06.05--h589041f_9' }"
+
+    input:
+    tuple val(meta), path(fasta)
+    val(sequencing_system)
+    val(fold_coverage)
+    val(read_length)
```

Comment:
These are all mandatory arguments?

Reply:
The documentation is a bit poor - this is the minimal set of required parameters I figured out by testing!

Reply:
Allright they can stay then :)